components/director/internal/domain/runtime/service.go

```diff
@@ -140,6 +141,10 @@ func (s *service) GetByTokenIssuer(ctx context.Context, issuer string) (*model.R
 	return runtime, nil
 }
 
+func (s *service) GetByFiltersGlobal(ctx context.Context, filter []*labelfilter.LabelFilter) (*model.Runtime, error) {
```

Comment:
Unit testing?

Reply:
Is it really necessary to write such a test? What I think it would do is simply test that error from repo layer is forwarded which is obvious. Is there any other reason for writing such a test, that I am missing?